shell/platform/darwin/ios/ios_surface.mm

```diff
@@ -76,11 +67,7 @@ bool IsIosEmbeddedViewsPreviewEnabled() {
 }
 
 ExternalViewEmbedder* IOSSurface::GetExternalViewEmbedderIfEnabled() {
```

Comment:
nit: should we just remove GetExternalViewEmbedderIfEnabled?

Reply:
This is the common method that's called by all the different surface implementations. So it is better to keep it there. Or we can update the method name (removing the `IfEnabled`)?

Reply:
Yup. `GetExternalViewEmbedder` mirrors Android's.